### PR TITLE
Add car collection UI

### PR DESCRIPTION
**Pull request by kennyintao**

### CODE REVIEW

**Comment by kennyintao** [comment]
image

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+  { name: 'Van', image: vanImage },
+  { name: 'Volvo', image: volvoImage },
+  { name: 'Tesla', image: teslaImage },
+  { name: 'RV', image: rvImage }
+];
+
+const Home: React.FC = () => {
+  return (
+    <div className="container mx-auto px-4">
+      <h2 className="text-2xl font-bold text-gray-800 my-8">Car models you might like</h2>
```
Comment (Hannibal0416) [inline src/pages/Home.tsx:29]: For the landing page, we have some changes, we will have two sections, the first section is car brands, and the section is car types, the example as below:
`<h2> Car brands you might like</h2>
<div/>
<h2> Car types you might like </h2>
<div/>
`
the modes will be shown on the next page regarding the user has chosen which brand or type.

Comment (Hannibal0416) [reply src/pages/Home.tsx:29]: BTW, the image urls are stored in the backend, but you can keep this way for our demo.
we will discuss it in the meeting on Wednesday.
Thanks!

File: yarn.lock
Comment (Hannibal0416) [inline yarn.lock:1]: this file should be added into .gitignore

**Pull request merged**